README.md

```diff
@@ -172,34 +194,137 @@ On iOS installation is a bit more complicated, because you need to setup a scree
    Replace `{{GROUP_IDENTIFIER}}` and `{{BUNDLE_IDENTIFIER}}` with your group identifier and bundle identifier respectively.
 9. Rebuild the app and enjoy!
 
-> **Note:** If the build fails due to sandbox issues (like `realpath`'s illegal option), you can disable sandboxing for
-> the extension target. To do this, open Xcode, go to the `MembraneScreenBroadcastExtension` target settings, select
-> `Build Settings` tab and disable `User Script Sandboxing`.
+# Example
 
-## Docs
+We strongly recommend checking out our example app that implements a basic video room client. To run the app:
 
-API documentation is available [here](https://jellyfish-dev.github.io/react-native-client-sdk/)
+1. Go to Membrane's server demo repo: https://github.com/membraneframework/membrane_videoroom. Follow instructions there to setup and run demo server.
+2. Clone the repo
+3. ```
+   $ cd example
+   $ yarn
+   ```
+4. In App.ts replace server url with your server's url.
+5. `yarn run android` or `yarn run ios` or run project from Android Studio / Xcode just like every RN project. Note that simulators won't work, you have to test on real device for camera and screensharing to run.
+
+# Usage
 
-## Usage
+Important note!! Since version 7.4.0 call function `initializeWebRTC()` once in your app before using any other functionality.
```

Comment:
```suggestion
> [!IMPORTANT]  
> Since version 7.4.0, you need to call function `initializeWebRTC()` once in your app before using any other functionality.
```